src/Subscriptions/SubscriptionPhase.php

```diff
@@ -226,7 +225,7 @@ public function get_next_date() {
 	 * @return void
 	 */
 	public function set_next_date( $next_date ) {
-		$this->next_date = ( null === $next_date ) ? null : DateTimeImmutable::create_from_interface( $next_date );
+		$this->subscription->set_next_payment_date( $next_date );
```

Comment:
Calling `$phase->set_periods_created()` will result in updating the subscription next payment date. Is that ok? We're not using the periods created, but maybe still available in older subscriptions / updating from older plugin version?

https://github.com/pronamic/wp-pay-core/blob/5677c598542ab9c89fc8ed0643f3644232486355/src/Subscriptions/SubscriptionPhase.php#L592-L594

https://github.com/pronamic/wp-pay-core/blob/5677c598542ab9c89fc8ed0643f3644232486355/src/Subscriptions/SubscriptionPhase.php#L315-L323

Reply:
Yes, that is ok, it will call `$phase->set_next_date()` and that will call `$this->subscription->set_next_payment_date( $next_date );`.